README.md

```diff
@@ -10,9 +26,9 @@ sudo pip install mkdocs-material markdown-include
 mkdocs serve --dev-addr 0.0.0.0:8000
 ```
```

Comment:
This section can be changed to tell the user to fork the project instead of cloning the main repo.